rdrf/registry/patients/models.py

```diff
@@ -756,19 +758,38 @@ def hard_delete_clinicaldata(self, patient_id):
         for cd in clinicaldata_models:
             cd.delete()
 
-    def set_consent(self, consent_model, answer=True, commit=True):
+    def notify_consent_change(self, consent_value):
+        if self.clinician and not consent_value.answer:
+            logger.info(f"Notify clinician {self.clinician} for change on {consent_value.consent_question}")
```

Comment:
Drop the clinician part from the log message. We're notifying everyone configured...

Reply:
Updated.